public/manifest.json

```diff
@@ -7,22 +7,9 @@
   "icons": {
     "128": "extension_icon.png"
   },
-  "content_scripts": [
-    {
-      "matches": [
-        "https://www.youtube.com/*",
-        "https://m.youtube.com/*"
-      ],
-      "js": [
-        "assets/extension.js"
-      ],
-      "css": [
-        "assets/extension.css"
-      ],
-      "run_at": "document_end",
-      "all_frames": true
-    }
-  ],
+  "background": {
```

Comment:
❓ I'm thinking about pros/cons of the service worker approach. Pros – we have a point to control extension injection, cons – we require more permissions and more logic to maintain (injection code).

If I'm not mistaken, we'll have to do a page reload when the extension is disabled/enabled anyways (since parts of the DOM were removed via JS). 

I was thinking about not adding extra code and simply do a full page refresh in the `Settings.vue` component after we toggle enabled/disabled option. Wdyt?